File: DesktopApp/Model/Syllabus.cs
using Newtonsoft.Json;

namespace ProfessorPerformanceEvaluation.Model
{
    public class Syllabus
    {
        [JsonProperty("idSyllabus")]
        public int IdSyllabus { get; set; }
        [JsonProperty("idEducationalProgram")]
        public int IdEducationalProgram { get; set; }
        [JsonProperty("idEducationalExperience")]
        public int IdEducationalExperience { get; set; }
    }
}

File: MobileApp/app/src/main/java/com/example/professorperformanceevaluation/model/Syllabus.java
package com.example.professorperformanceevaluation.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class Syllabus implements Serializable {

    @SerializedName("idSyllabus")
    @Expose
    private int idSyllabus;
    @SerializedName("idEducationalProgram")
    @Expose
    private int idEducationalProgram;
    @SerializedName("idEducationalExperience")
    @Expose
    private int idEducationalExperience;

    public Syllabus() {
    }

    public int getIdSyllabus() {
        return idSyllabus;
    }

    public void setIdSyllabus(int idSyllabus) {
        this.idSyllabus = idSyllabus;
    }

    public int getIdEducationalProgram() {
        return idEducationalProgram;
    }

    public void setIdEducationalProgram(int idEducationalProgram) {
        this.idEducationalProgram = idEducationalProgram;
    }

    public int getIdEducationalExperience() {
        return idEducationalExperience;
    }

    public void setIdEducationalExperience(int idEducationalExperience) {
        this.idEducationalExperience = idEducationalExperience;
    }

}